cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -31,23 +36,32 @@
                     }
                 ]
             }
-        %}
+        -%}
         {# fmt:on #}
     {% else %}
-        {% set contact_details = "" %}
+        {%- set contact_details = "" -%}
     {% endif %}
 
     {% if page.is_latest %}
-        {% set release_html -%}
-            <a href="{{ routablepageurl(page.get_parent().specific, "previous_releases") }}">{{ _("View previous releases") }}</a>
-        {%- endset %}
+        {% if latest_version_url %}
+            {%- set infoPanelText = _("This version has been amended since publication") -%}
+            {%- set releases_text = _("View amended version") -%}
+            {%- set infoPanelType = 'ons-red' -%}
+            {%- set releases_url = latest_version_url -%}
+        {% else %}
+            {%- set infoPanelText = _("Latest release") -%}
+            {%- set releases_text = _("View previous releases") -%}
+            {%- set infoPanelType = 'ons-green' -%}
+            {%- set releases_url = routablepageurl(page.get_parent().specific, "previous_releases") -%}
+        {% endif %}
     {% else %}
-        {% set release_html -%}
```

Comment:
as noted in Slack, this is still used by the hero, resulting in an empty "Releases:" item

Reply:
Resolved, removed this and the editions meta data which is no longer needed/used by designs :)